client/profile-wizard/steps/business-details/flows/selective-bundle/index.js

```diff
@@ -125,12 +167,37 @@ class BusinessDetails extends Component {
 		];
 
 		if ( businessExtensions.length ) {
+			const installationStartTime = window.performance.now();
 			promises.push(
 				installAndActivatePlugins( businessExtensions )
 					.then( ( response ) => {
+						const totalInstallationTime =
+							window.performance.now() - installationStartTime;
+						const installedExtensionsData = prepareExtensionTrackingInstallationData(
+							extensionInstallationOptions,
+							response
+						);
+
+						recordEvent(
+							'storeprofiler_store_business_features_installed_and_activated',
```

Comment:
Just noting that Google and WCPay will always be listed here with the current logic, even when they are already installed/activated.  Not sure if that will be misleading with the data since the timeframes will be shorter.
![Screen Shot 2021-12-27 at 3 00 43 PM](https://user-images.githubusercontent.com/10561050/147504208-afaa1c7c-f2ad-48f6-abe6-6cc9af21f36a.png)

Reply:
I updated the logic, so we don't call the install and activate hook when all the selected extensions are already activated and installed. This will prevent the above scenario. The changes for this are here: https://github.com/woocommerce/woocommerce-admin/pull/8042/commits/6e47c4cb4a3cd541c44214b042d758b5f1cbe5a4